cookbooks/fb_collectd/libraries/default.rb

```diff
@@ -0,0 +1,50 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+#
+# Copyright (c) 2016-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+module FB
+  # collectd utility functions
+  class Collectd
+    # Internal helper function to generate /etc/collectd/collectd.conf entries
+    def self._gen_collectd_conf_entry(k, v, i = 0)
+      indent = ' ' * i
+      # For keys with multiple values (i.e. an array for a value) we need to
+      # specify the whole line multiple times, so we generate:
+      #   foo = "key1"
```

Comment:
this is confusing (I know, I wrote it but now I've read it). That should be:

  key = "foo1"
  key = "foo2"

:)